jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/ResponseHeaderRegexRule.java

```diff
@@ -0,0 +1,138 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.rewrite.handler;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.eclipse.jetty.http.HttpStatus;
+import org.eclipse.jetty.server.Response;
+import org.eclipse.jetty.util.Callback;
+import org.eclipse.jetty.util.StringUtil;
+
+/**
+ * Sends the response code whenever the rule finds a match.
```

Comment:
This should be clearer:

```suggestion
 * Sends a response with the configured status code whenever the value of the configured request header matches a regular expression.
```

Reply:
Also based on ResponsePatternRule.java, it was on my todo list to clean up after the code had a once over. Thanks for the suggestion, I updated it.